build.gradle

```diff
@@ -57,6 +57,7 @@ task removeUnsupported(type: Delete) {
   dependsOn extractRelease
   delete "${jarDir}/${pkgDir}/aix-ppc"
   delete "${jarDir}/${pkgDir}/aix-ppc64"
+  delete "${jarDir}/${pkgDir}/linux-arm"
```

Comment:
I see, the GitHub UI is playing a trick on me then because at first I didn't see it and then I came back to see if you commented about our version discussion on elastic/elasticsearch#25318 and then I saw it so I assumed I must have only seen a subset of changes on my first review but now I realize it was on the second visit that somehow I only saw a subset. So sorry for the noise.